src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -57,7 +57,7 @@ module.factory('SavedDashboard', function (courier, config) {
     timeRestore: 'boolean',
     timeTo: 'string',
     timeFrom: 'string',
-    refreshInterval: 'string'
+    refreshInterval: 'object'
```

Comment:
I think we also need to create mappings for refreshInterval's properties, otherwise there's a risk they'll get mapped incorrectly if we rely on dynamic mapping. I'm not sure if there's an example of this already being done or not.